File: lab/javascript/pares/poui.js
function par_ou_impar(numero){
    return numero % 2 == 0;
}


const prompt = require('prompt-sync')();
numero = Number(prompt());

if (par_ou_impar(numero)){
    console.log("Par");
} else{
    console.log("Impar");
}

File: lab/c/pares/poui.c
#include <stdio.h>


int par_ou_impar(int numero){
    return numero % 2 == 0;
}


int main(){
    int numero;
    scanf("%d", &numero);

    if (par_ou_impar(numero)){
        printf("Par\n");
    } else{
        printf("Impar\n");
    }
}
